small typos, added TOC, correction jumbling link

evezeyl opened the pull request:
This seems to present ok.
The filtering of variant is actually the step that is detailed - where thought to present file. Maybe we should move that up ?

Thomieh73 commented on docs/source/tutorials/bcftools.md:
```diff
@@ -44,16 +66,17 @@ Some examples of what you can do:
 It also provide some utilities for files manipulations, and can provide statistics on variant calls, and much more. It seems eg. that it can help determining Hq variants by allowing to extract variants congruent between different variants callers (intersection).
 
+---
 ### Ressources
```
One s to much in "Resources"

evezeyl replied:
ok

Thomieh73 commented on docs/source/tutorials/bcftools.md:
```diff
@@ -267,7 +294,7 @@ in the tutorial data`/cluster/projects/nn9305k/tutorial/20240226_bcftools/` subf
 3. We recreate the consensus
 4. We compare our consensus to Snippy consensus.
-Because in theory those sequences should be identical (obtained by the same means, but it might have been some headers reformating), we can concatenate "our" consensus and Snippy consensus and pretend this is an alingnment and detect SNPs differences between those consensus files. 
+Because in theory those sequences should be identical (obtained by the same means, but it might have been some headers reformating), we can concatenate "our" consensus and Snippy consensus and pretend this is an alingnment and detect SNPs differences between those consensus files.
```
type in the word " alingnment", remove first "n"

Thomieh73 reviewed (changes requested):
Hi @evezeyl , I just found a few little typo's. Change those and then finish this request

evezeyl commented on docs/source/tutorials/bcftools.md:
```diff
@@ -24,14 +45,15 @@ Taking control of the variant calling process and how variant files are manipula
 Snippy do not use bcftools for variant calling[^3], but it uses it for several purposes:
 filtering variants, creating consensus, converting, compressing and indexing variant files.
-Bbcftools offers a variety of commands/modules to manipulate VCF files. Combining those in different way,
+Bcftools offers a variety of commands/modules to manipulate VCF files. Combining those in different way,
```
test

evezeyl commented on docs/source/tutorials/bcftools.md:
```diff
@@ -44,16 +66,17 @@ Some examples of what you can do:
 - Detect the number of chromosomal copies (polysomy)
 
-It can also work working with haplotypes, determine consequences of variations (eg Frame shift), and in association to specific mutation annotations (eg. as with [SnpEff]) as done in [Snippy] it can eg. help determine the effects of a mutation.
+It can also work working with haplotypes, determine consequences of variations (eg Frame shift), and in association to specific mutation annotations (eg. as with [SnpEff]) as done in [Snippy] then it can eg. help determine the effects of a mutation.
```
for me the then does not fit ... otherwise working removed

evezeyl commented:
Damned ... did not want to ask 2 reviewers @Thomieh73 you do not need - I updated and re-read fast. Answered, and fixed layout. I think we can merge - although this tutorial can be continued but be in place already as is
